src/services/recoveryVaults/format.ts

```diff
@@ -107,7 +107,42 @@ export default ({
         value: identity?.userAssignedIdentities[key],
       })),
     },
-    ...transformSystemData(systemData),
```

Comment:
reason for this change? Also we should avoid using spread because if additional properties are added they wont be in the schema (maybe not an issue if we lock down the azure sdk version)